examples/mobile/build.gradle

```diff
@@ -0,0 +1,25 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+plugins {
+	id 'com.android.application' version '8.0.0' apply false
+	id 'com.android.library' version '8.0.0' apply false
+}
+
+task clean(type: Delete) {
+    delete rootProject.buildDir
+}
```

Comment:
I don't understand this task. Isn't this already the default behaviour of the clean task?

Reply:
I added the other required cleanup into the task.